docs/ref/models/class.txt

```diff
@@ -11,6 +11,35 @@ reference guides </ref/models/index>`.
 Attributes
 ==========
 
+``DoesNotExist``
+----------------
+
+.. exception:: Model.DoesNotExist
+
+    This exception is raised by the ORM in a couple places, for example by
+    :meth:`QuerySet.get() <django.db.models.query.QuerySet.get>` when an object
+    is not found for the given lookups.
+
+    Django provides a ``DoesNotExist`` exception as an attribute of each model
+    class to identify the class of object that could not be found and to allow
+    you to catch a particular model class with ``try/except``. The exception is
```

Comment:
`…class of object that could not be found allowing capture of the exception for that specific class with ``try/except``.`?

Reply:
Yes that's nicer chaining.
(Please use github suggestions for such edits, makes it easier to merge.)

Reply:
I'm sticking with "allowing you to" because using the "you" voice makes the docs more friendly imo.